### PR TITLE
Add a book for copyright and licensing.

### PR DESCRIPTION
**Pull request by SethiPandi**
## What does this PR do?
Add Resource(s)

### Description 
Adding a book to the license section which helped a lot with understanding copyright and open-source software licensing.

### Why is this valuable (or not)
* It does a good job explaining in layman's terms the difference between copyright and license.
* Explains what copyright actually is with simple analogies to explain how it works, and what the public domain is.
* Runs through some of the most popular open-source licenses such as Apache and GPL clause by clause to explain what it means.

### How do we know it's really free?
It's published by a reputable party in paper back form and available under the O'Reilly Open Book platform in digital form.  
It's published under the Creative Commons BY-ND 2.0 license.

### For book lists, is it a book?
Yes

### Checklist:
- [x] Not a duplicate
- [x] Included author(s) if appropriate
- [x] Lists are in alphabetical order
- [x] Needed indications added (PDF, access notes, under construction)

### CODE REVIEW

**Comment by eshellman** [comment]
a bit old and scruffy. Is this the best version available?

**Comment by SethiPandi** [comment]
I contacted the author of the book prior to this PR to verify the "official" source, as I originally found the book from a PDF via search engine.

The author directed me to this page.

Unfortunately the lack of styling on the website appears to be a bug or issue on O'Reillys end; I've dropped them an email about it to report the problem already.

**Comment by SethiPandi** [comment]
On in fact they responded to my email already, and said they've filed a report and will fix the issue.

**Comment by eshellman** [comment]
also, since it's CC licensed, you can make your own ebook of it!

**Comment by SethiPandi** [comment]
Sorry, I may have misunderstood something then.
When you said "old and scruffy", were you referring to the website or the book itself? (I assumed website.)

If the book is the issue, I'll gladly compile all sections together into a single document. O'Reilly decided to be annoying and split it up to one download per chapter. Are there any other concerns than that?

**Comment by eshellman** [comment]
If you think it's worth the trouble to clean it up, by all means we'll add it. A better copy might be on Unglue.it or archive.org, and if not you can add one there. Also check the author's website. We had to remove a lot of O'Reilly titles when they stopped promoting open content.
